code_challenges.rb

```diff
@@ -33,19 +45,60 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
-
-
+# create method takes a string as an argument
+def no_vowels string 
+# removes all vowels from string
+        # string.downcase.delete "aeiou"
+        string.delete "aeiouAEIOU"
+end
+p no_vowels(album1)
+p no_vowels(album2)
+p no_vowels(album3)
 
 
 # --------------------4) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
-
-
-
+# create a class called Bike
+class Bike
+    # create initialize method
+    def initialize(model, wheels, current_speed)
+        @model = model
+        @wheels = 2
+        @current_speed = 0
+    end
+
+    def pedal_faster speed
+        @current_speed += speed
+    end
+
+    def brake speed
+        @current_speed -= speed
+        if @current_speed - speed < 0
+            @current_speed = 0
+        else
+            @current_speed = speed
+        end
+    end
+
+    def get_info
+        puts "My bike is a #@model it has #@wheels wheels and is going #@current_speed mph"
+    end
+end
+
+my_bike = Bike.new("Trek", @wheels, @current_speed) 
```

Comment:
instance variables should only be used within a class (within an instance)... because this variable is getting declared outside of the class, `@wheels` and `@current_speed` have no reference to the instance variables created within the class